app/src/main/res/values/colors.xml

```diff
@@ -1,8 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <!-- Add your application custom colors here -->
-    <!-- Base application colors -->
+    <!-- General colors-->
     <color name="colorPrimary">#1D9BD7</color>
     <color name="colorPrimaryDark">#0960B7</color>
     <color name="colorAccent">#8DC63F</color>
+    <!-- Login colors -->
+    <color name="white">#FFFFFF</color>
+    <color name="green">#8bd62f</color>
+    <color name="green_transparent">#CC8bd62f</color>
+    <color name="blue">#288dd6</color>
+    <color name="blue_transparent">#CC288dd6</color>
```

Comment:
This is good, but is not that good (sad-guido-kaczka-crying).  For sure, you are going to use those colors in almost every view. Its a good idea to "abstract" those colors names to concepts. For example:
* green and blue are the main and the secondary color of the app
* green_transparent and blue_transparent are only for the gradient
* white is a android base color. 
So, I would change these things like this:
* green -> wolox_green (useful in every view with the same green)
* blue -> wolox_blue
* green_transparent -> gradient_logo_green
* blue_transparent -> gradient_logo_blue
* white -> use `android:colors/white`